site/content/integrate/apps/api/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "API"
+heading: "Mattermost Apps API Reference"
+description: "This reference describes the Mattermost Apps API"
+weight: 200
+---
+
+## App Structure
+
+Apps consist of just 3 things:
+- [manifest.json]({{< ref "manifest" >}}) file
+- [Call]({{< ref "call" >}}) responders, starting with `/bindings` (see [Interactivity]({{< ref "interactivity" >}})) and `/on_install` (see [Manifest]({{< ref "manifest" >}}))
+- Static assets, currently just icon files
+
+## Mattermost Authentication
+An app declares in its `Manifest` the `Locations` that it will be binding to, and `Permissions` it will require to operate. These are consented to by the system administrator when installing Mattermost apps interactively, in Mattermost.
+
+Each app when installed into Mattermost gets an OAuth2 cientID/secret (not yet used), and a Bot User Account with a personal access token. Each call may receive a combination of `bot_access_token`, `acting_user_access_token`, and `admin_access_token` as applicable. See [Authentication]({{< ref "authentication" >}}) for more.
+
+Apps can use OAuth2 to connect to 3rd party APIs. See [3rd party OAuth2]({{< ref
+"3rdparty-oauth2" >}}) for more.
+
+
+
+
+Each call request an app receives contains a `bot_access_token` in the `context`. 
+
+Additionaly, if the app was granted `act_as_user` permission, and the call's
+`expand` contained `acting_user_access_token=all`, the call receives
+`acting_user_access_token` in the request `context`.
```

Comment:
0/5 - s/"in the request context"/"in the context" (to be consistent with the other two paragraphs).